dom/telephony/gonk/TelephonyService.js

```diff
@@ -361,6 +361,7 @@ function TelephonyService() {
   this._currentConferenceState = nsITelephonyService.CALL_STATE_UNKNOWN;
   this._audioStates = [];
   this._ussdSessions = [];
+  this._ussdCallbacks = [];
```

Comment:
An array of USSD callbacks (one per SIM card).
This corresponds to the latest promise registered